Makefile2

```diff
@@ -0,0 +1,46 @@
+UPSTREAM=https://github.com/ggerganov/llama.cpp.git
+WORKDIR=llama/vendor
+FETCH_HEAD=46e3556e01b824e52395fb050b29804b6cff2a7c
+
+all: sync
```

Comment:
Assuming this will remain vendoring only logic, perhaps a better filename is something like `vendor.make` and maybe the default target would give some usage instructions.

I was going to suggest renaming this to `Makefile` but since it's not used to actually build things, I think that will lead to confusion for new developers checking out the repo - if there's a makefile, a good guess is just type `make` and it'll build.

Another option would be to move this out of the top of the tree since most people building from source will never do vendoring updates.

Reply:
Yes, if this stays as a root level `Makefile` it should also build the project, ideally with GPU support